### PR TITLE
Add a GetObjCount method

### PR DESCRIPTION
**Pull request by jojimt**
This commit adds an obj count method to each model object.

### CODE REVIEW

File: generators/templates.go [hunk 1]
```diff
@@ -402,6 +402,14 @@ var {{ initialCap .Name }}ModalView = React.createClass({
 module.exports.{{ initialCap .Name }}SummaryView = {{ initialCap .Name }}SummaryView
 module.exports.{{ initialCap .Name }}ModalView = {{ initialCap .Name }}ModalView
   `,
+	"objcount": `
+{{ range .Objects }} {{ if .CfgProperties | len }}
+func Get{{ initialCap .Name }}ObjCount() int {
```
Comment (erikh) [inline generators/templates.go]: Maybe just call this `GetXCount`, as it's a little less verbose?

**Comment by erikh** [comment]
code LGTM

File: generators/templates.go (hunk 1 above)
Comment (jojimt) [reply generators/templates.go]: You mean omit "Obj" and make it look like GetTenantCount()?

**Comment by erikh** [comment]
Yeah.

**Pull request merged**